CMakeLists.txt

```diff
@@ -9,12 +9,20 @@ if(NOT FORCE_BUILD_VENDOR_PKG)
   find_package(ignition-cmake2 QUIET)
 endif()
 
+# CMAKE_BUILD_TYPE is not used by MSVC, we need to use the variable somehow
+# to avoid the warning
+if (MSVC)
+  set(ignore ${CMAKE_BUILD_TYPE})
+endif()
+
```

Comment:
This can be dropped. The warning is coming from the external project, not the vendor project.